docs/en_US/guides/sideloading-apps.md

```diff
@@ -52,24 +53,35 @@ Sideloadly is compatible with iOS 7 and newer.
 1. Plug your iOS device into your computer
     - Make sure your computer is trusted and allowed to view the contents of your device
 1. Drag and drop the `.ipa` file of your choice into Sideloadly
-1. Enter in your Apple ID
+1. Enter your Apple ID
 1. Enter in your password
-    - Sideloadly must make a request to it's servers in order to work with free developer accounts. If you are not OK with this, you may use an alternate Apple ID.
+    - Sideloadly must make a request to its servers in order to work with free developer accounts. If you are not OK with this, you may use an alternate Apple ID.
 
 ### Sideload using AltStore
 
-<!--I will add AltStore 1.5 soon don't worry-->
 
 ::: tip
 
-AltServer is only compatible with iOS 12.2 and newer.
+AltStore is only compatible with iOS 15 and newer.
+
+AltServer for macOS is only compatible with 11.0 and higher.
+(10.14 and 10.15 can download [AltServer 1.6.2](https://f000.backblazeb2.com/file/altstore/altserver/1_6_2.zip))
```

Comment:
I just checked and this link is legit, and is referenced on https://faq.altstore.io/getting-started/how-to-install-altstore-macos. Just thought I'd mention that